modules/sql/src/main/scala/SqlModule.scala

```diff
@@ -14,8 +14,24 @@ trait SqlModule[F[_]] {
   /** The type of a codec that reads and writes column values of type `A`. */
   type Codec[A]
 
+  /** A codec that has forgotten its type argument. */
```

Comment:
`Codec[_]` isn't a type in Scala 3 so we have to push the existential down into a type member.